README.rst

```diff
@@ -11,41 +11,62 @@ The tool will also notify release managers about potential issues in bugzilla an
 The list of checkers is documented on the Mozilla wiki:
 https://wiki.mozilla.org/BugBot
 
-
 This package currently uses Mozilla's `Bugzilla REST API <https://wiki.mozilla.org/Bugzilla:REST_API>`_, and the Mozilla IAM `phonebook <https://github.com/mozilla-iam/cis/blob/master/docs/PersonAPI.md>`_ (to access bug assignees' managers & Mozilla email addresses).
 
 
 Installation
 ------------
 
+bugbot can run either using `uv <https://docs.astral.sh/uv/>`_ to manage the Python environment,
+or with manual management of the virtual environment.
+
 #. Check out the code::
 
     git clone https://github.com/mozilla/bugbot.git
 
-#. (optional) Create your virtualenv using virtualenvwrapper::
+#. If not using ``uv``:
 
-    virtualenv -p python3 venv
-    source venv/bin/activate
+  #. (optional) Create your virtualenv using virtualenvwrapper::
 
-#. Install the dependencies for Python 3 too::
+      virtualenv -p python3 venv
+      source venv/bin/activate
 
-    pip install -r requirements.txt
+  #. Install the dependencies for Python 3 too::
+
+      pip install -e .
```

Comment:
A convenient aspect of `uv` is that it simplifies the process. We don’t need to explain how to set up venv or even specify a Python version, as it’s already declared in the `pyproject.toml` file. So we can simplify this section accordingly.


```suggestion
#. Install `uv <https://docs.astral.sh/uv/getting-started/installation/>`_

#. Check out the code::

   git clone https://github.com/mozilla/bugbot.git

#. Install the dependencies::
   uv sync --frozen --all-extras --dev
```

Reply:
If we're happy to go all-in on `uv` I don't think you need to run `uv sync` manually in dev; it's fine to use `uv run` later.